public/static/docs/command-reference/gc.md

```diff
@@ -42,16 +46,20 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
   useful if tags are used to track "checkpoints" of an experiment or project.
   Note that both options can be combined, for example using the `-aT` flag.
 
+- `-w`, `--workspace` - remove files in local cache. _This operation is
```

Comment:
may be I'm missing something, but `workspace` is not the opposite to `-c`. It's one of the options (along with `-a` and `-T`) to provide the scope of files we would like to keep (and remove everything else). So this option description should be similar to `-a` and/or `-T`

Reply:
`-w` triggers the same behavior as plain-old `gc`, in that it deletes thing (unless `-a`/`-T` is specified). It's similar to `-c` option..

Reply:
from the description it sounds like it does delete all files in cache. This is not true (I hope). We need to describe it somehow. The best to describe is to use "keep" semantics like in `-a` or `-T` (i.e. what scope of DVC-file references we keep and remove everything else in cache).

Also, I hope I can run just `-a` still, right? Or just `-T`?

Reply:
Ahh, okay. I misunderstood you there.

Reply:
Yes, you can just do `-a` or `-T` without mentioning `-w`.